test/Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM us.gcr.io/logdna-k8s/node:14-ci as base
```

Comment:
this isn't accessible to anyone out side the org. So anyone wanting to contribute would not be able to run the test suite

Reply:
Ah thanks, habit.